src/hb-directwrite.cc

```diff
@@ -928,3 +942,15 @@ hb_directwrite_face_create (IDWriteFontFace *font_face)
   return hb_face_create_for_tables (reference_table, font_face,
 				    _hb_directwrite_font_release);
 }
+
+/**
+* hb_directwrite_face_get_font_face:
+* @face:
+*
+* Since: REPLACEME
+**/
+IDWriteFontFace *
+hb_directwrite_face_get_font_face (hb_face_t *face)
+{
+  return face->data.directwrite->fontFace;
```

Comment:
What if `data` is null or it isn’t a DirectWrite-backed face?

Reply:
IIUC it will always be initiated as can be seen on the test code. I can do null check but will be nice to apply such to `hb_coretext_face_get_cg_font` also.

Reply:
Shouldn't need check.

Reply:
Hmm, I guess we should think about the naming also